src/components/DefaultConfig.js

```diff
@@ -20,7 +21,8 @@ export const defaultConfig = {
     fullScreen: false,
     parsers: [
         KatexParser,
-        HighlightjsParser
+        HighlightjsParser,
+        InjectLnParser
```

Comment:
`InjectLnParser` Should be import at MarkdownPalettes.vue

Reply:
Why? We also import `HighlightjsParser` and `KatexParser` in this file.

Reply:
Because Content Parser not only be used in editor, it can also be export as a function.

Reply:
But `defaultConfig` is just the config of the editor, right? e.g. `defaultConfig.fullScreen`. I don't think it will be used in other places.

Reply:
But I think get default config from `defaultConfig` is more properly.

Reply:
Generally, contentParser and Editor use same rendering config.But `InjectLnParser` only used in editor

Reply:
Of course, `getDefaultContentParser` in `src\module` need update.

Reply:
OK, I have understood it.